PCLConfig.cmake.in

```diff
@@ -93,13 +93,8 @@ macro(find_boost)
   elseif(NOT BOOST_INCLUDEDIR)
     set(BOOST_INCLUDEDIR "@Boost_INCLUDE_DIR@")
   endif()
-  
-  set(Boost_ADDITIONAL_VERSIONS
-    "@Boost_MAJOR_VERSION@.@Boost_MINOR_VERSION@.@Boost_SUBMINOR_VERSION@" "@Boost_MAJOR_VERSION@.@Boost_MINOR_VERSION@"
```

Comment:
What about this line? This would be lost. Maybe something like
```cmake
set(Boost_ADDITIONAL_VERSIONS "@Boost_MAJOR_VERSION@.@Boost_MINOR_VERSION@.@Boost_SUBMINOR_VERSION@" "@Boost_MAJOR_VERSION@.@Boost_MINOR_VERSION@" ${Boost_ADDITIONAL_VERSIONS})
```

Reply:
But the Boost_MAJOR_VERSION etc. refers to the boost that PCL was build with - so that should already be in that original list? 

but maybe it should just be that one that stays or is the intention that you fx. build PCL with boost 1.75.0 and then use another boost version in the downstream project? Wouldn't make that much sense?

Reply:
This logic was added in https://github.com/PointCloudLibrary/pcl/pull/1630 . If I understand correctly, it exists so that CMake prefers the Boost version that PCL was built with.

Reply:
Yeah, I guess we can keep it that way.